### PR TITLE
[5.0] Simplify boolean logic

### PR DESCRIPTION
**Pull request by Denitz**
### Summary of Changes

Simplify boolean logic.

### Testing Instructions

Apply patch

### Actual result BEFORE applying this Pull Request

Bad code

### Expected result AFTER applying this Pull Request

Nice code

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Review by HLeithner — APPROVED** [review]
Needs a second pair of eyes here

File: administrator/components/com_menus/src/Field/Modal/MenuField.php [hunk 1]
```diff
@@ -152,8 +152,8 @@ public function setup(\SimpleXMLElement $element, $value, $group = null)
 
             // Creating/editing menu items is not supported in frontend.
             $isAdministrator = Factory::getApplication()->isClient('administrator');
-            $this->allowNew  = $isAdministrator ? ((string) $this->element['new']) === 'true' : false;
-            $this->allowEdit = $isAdministrator ? ((string) $this->element['edit']) === 'true' : false;
+            $this->allowNew  = $isAdministrator && ((string)$this->element['new']) === 'true';
+            $this->allowEdit = $isAdministrator && ((string)$this->element['edit']) === 'true';
```
Comment (Quy) [inline administrator/components/com_menus/src/Field/Modal/MenuField.php]:
```suggestion
            $this->allowNew  = $isAdministrator && (string) $this->element['new'] === 'true';
            $this->allowEdit = $isAdministrator && (string) $this->element['edit'] === 'true';
```

File: plugins/system/accessibility/src/Extension/Accessibility.php [hunk 2]
```diff
@@ -89,7 +89,7 @@ public function onBeforeCompileHead()
                             'units' => 'px',
                         ],
                     ],
-                    'useEmojis' => $this->params->get('useEmojis') != 'false' ? true : false,
+                    'useEmojis' => $this->params->get('useEmojis') != 'false',
```
Comment (Quy) [inline plugins/system/accessibility/src/Extension/Accessibility.php:92]:
```suggestion
                    'useEmojis' => $this->params->get('useEmojis') !== 'false',
```

Comment (Quy) [inline plugins/system/accessibility/src/Extension/Accessibility.php]:
```suggestion
                    'useEmojis' => $this->params->get('useEmojis') === 'true',
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 0f0087efaf171c1f41a45fe8f8af882065a86a2b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41655">issues.joomla.org/tracker/joomla-cms/41655</a>.</sub>

**Comment by Quy** [comment]
I assume you are referring to this one:

`'useEmojis' => $this->params->get('useEmojis') === 'true',`

Yes, I toggled its setting and it worked accordingly.

File: libraries/src/Language/Text.php [hunk 3]
```diff
@@ -59,7 +59,7 @@ public static function _($string, $jsSafe = false, $interpretBackSlashes = true,
                 $script = (bool) $jsSafe['script'];
             }
 
-            $jsSafe = \array_key_exists('jsSafe', $jsSafe) ? (bool) $jsSafe['jsSafe'] : false;
+            $jsSafe = \array_key_exists('jsSafe', $jsSafe) && $jsSafe['jsSafe'];
```
Comment (HLeithner) [inline libraries/src/Language/Text.php]:
```suggestion
            $jsSafe = !\empty($jsSafe['jsSafe']);
```

that should do the same and is easier to read

**Review by HLeithner — COMMENTED** [review]
Only one suggestion

**Comment by HLeithner** [comment]
Merging this for beta2, thanks

**Pull request merged**